e2e/binary_artifacts_test.go

```diff
@@ -173,5 +177,43 @@ var _ = Describe("E2E TEST:"+checks.CheckBinaryArtifacts, func() {
 			Expect(scut.ValidateTestReturn(nil, "binary artifacts", &expected, &result, &dl)).Should(BeTrue())
 			Expect(repoClient.Close()).Should(BeNil())
 		})
+		It("Should return binary artifacts present at commit in source code when using local repoClient", func() {
+			// create temp dir
+			tmpDir, err := ioutil.TempDir("", "")
+			Expect(err).Should(BeNil())
+			defer os.RemoveAll(tmpDir)
+
+			_, e := git.PlainClone(tmpDir, false, &git.CloneOptions{
+				URL: "http://github.com/ossf-tests/scorecard-check-binary-artifacts-e2e-4-binaries",
+			})
+			Expect(e).Should(BeNil())
+
+			dl := scut.TestDetailLogger{}
+			repo, err := localdir.MakeLocalDirRepo(tmpDir)
+			Expect(err).Should(BeNil())
+
+			x := localdir.CreateLocalDirClient(context.Background(), logger)
+			err = x.InitRepo(repo, "d994b3e1a8912283f9958a7c1e0aa480ca24a7ce")
```

Comment:
We don't have `commit` support in LocalRepo AFAIK. We need #1709 to be implemented for that. Keep this as `HEAD` for now.

Reply:
OK, got it. Thanks